### PR TITLE
Add-delete

### PR DESCRIPTION
**Pull request by andgarzonmal**
implemented the CRUD (create, update, delete) methods. All the elements of the user interface are fully functional

### CODE REVIEW

File: src/modules/add.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  if (arr.some((task) => task.description === inputForm.value)) {
+    alert('task already added');
+    return;
+  }
+
+  const task = {
+    index: arr.length,
+    description: inputForm.value,
+    completed: false,
+  };
```
Comment (ShoiraTa) [inline src/modules/add.js]: - [ ] By default,  the property `index` should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Which means that index should start from 1. 
You can simply do it by adding + 1 to array length 
`index: arr.length + 1`

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+**To get a local copy up and running follow these simple example steps**.
+
+### Prerequisites
+
+- You need to have a code editor to be able to run the code. VSCode is most preferred.
+
+### Setup
+
+- To clone the code press in the green button at the top right corner in this page
+- Select the method that you want to clone
```
Comment (ShoiraTa) [inline README.md:27]: - [ ] Kindly update these steps and include how one can setup the webpack, eg: run npm i/ npm run build / npm start. There are so many technologies and frameworks, people may not know how to get started with particular this project 😉

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @andgarzonmal ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Users can add tasks ✔️ 
- Users can delete tasks ✔️ 
- Users can update tasks ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] I have noticed that the package.json has no start script to run the application. I strongly recommend adding it, so that you can run `npm start` on terminal and it automatically opens project on browser listening to changes:
Screenshot 2022-06-01 at 22 49 50


_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+- Run npm to install dependencies on webpack => npm i
+- for you to see the actual project working in the browser run npm dev => npm run dev or npm run start
+- if you make some modifications to the project run npm run build in order to bundle your changes on dist folder => npm run build 
+
+### Author
+
+Andres Garzon Maldonado
+    
+  [GITHUB](https://github.com/andgarzonmal)
+  [LINKEDIN](https://www.linkedin.com/in/andres-garzon-maldonado-951a2a180/)
```
Comment (danmainah) [inline README.md:40]: - [ ] [OPTIONAL] It would be nice to add the issues section, allowing other developers to offer suggestions to improve the project.

**Review by danmainah — APPROVED** [review]
Hi @andgarzonmal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**